src/routes/news/components/NewsList.js

```diff
@@ -45,20 +51,26 @@ const HeaderImage: StyledComponent<{}, {}, *> = styled.Image`
   border-bottom-left-radius: 5px;
 `
 
-const Row: StyledComponent<{language: string}, ThemeType, *> = styled.View`
+const Row: StyledComponent<{ language: string }, ThemeType, *> = styled.View`
   flex-direction: ${props => contentDirection(props.language)};
   border-radius: 5px;
   width: 95%;
+  flex-wrap: wrap;
   align-self: center;
-  background-color: #0279a6;
+  padding: 5px;
+  background-color: ${props => props.theme.colors.tunewsThemeColor};
 `
 
-const ExtraInfo: StyledComponent<{ underlined?: boolean }, ThemeType, *> = styled.Text`
+const TunewsFooter: StyledComponent<
+  { underlined?: boolean, rightMargin: number },
```

Comment:
Is this correctly handled in RTL layouts? Probably yes?

Reply:
Yes, still will re-view UI again.

Reply:
Yes, it is but the problem that data comes in English not translated so it does not feel like it's handled :D